magik-checks/src/main/resources/nl/ramsolutions/sw/sonar/l10n/magik/rules/PragmaDoesNotIncludeTopicInTopics.html

```diff
@@ -1 +1 @@
-<p>Pragma does not contain all of the list of topics in topics.</p>
+<p>Pragma does not contain all topics from the list in topics.</p>
```

Comment:
What does "from the list mean"?

Reply:
I know, as I know that there is a parameter in the check. But somebody seeing this for the first time will not know.